admin/app/config.py

```diff
@@ -16,6 +16,7 @@ class Config(object):
     ADMIN_BASE_URL = os.environ.get('ADMIN_BASE_URL', 'http://localhost:6012')
 
     CDN_BASE_URL = os.environ.get('CDN_BASE_URL', 'https://notify-static-logos-staging.static.cld.gov.au')
+    DOCS_BASE_URL = os.environ.get('DOCS_BASE_URL', 'https://docs.notify.gov.au')
```

Comment:
Cool, you can now use the env var in templates? i.e. change hard-coded cases to use this config.

Reply:
I'll do it in a separate branch